### PR TITLE
Add diego recipe

### PR DESCRIPTION
**Pull request by jfallmann**
Describe your pull request here

This is a recipe for the DAS caller DIEGO
----

Please read the [guidelines for Bioconda recipes](https://bioconda.github.io/contributor/guidelines.html) before opening a pull request (PR).

* If this PR adds or updates a recipe, use "Add" or "Update" appropriately as the first word in its title.
* New recipes not directly relevant to the biological sciences need to be submitted to the [conda-forge channel](https://conda-forge.org/docs/) instead of Bioconda.
* PRs require reviews prior to being merged. Once your PR is passing tests and ready to be merged, please issue the `@BiocondaBot please add label` command.
* Please post questions [on Gitter](https://gitter.im/bioconda/Lobby) or ping `@bioconda/core` in a comment.

<details>
  <summary>Please use the following BiocondaBot commands:</summary>

Everyone has access to the following BiocondaBot commands, which can be given in a comment:

<table>
  <tr>
    <td><code>@BiocondaBot please update</code></td>
    <td>Merge the master branch into a PR.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please add label</code></td>
    <td>Add the <code>please review & merge</code> label.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please fetch artifacts</code></td>
    <td>Post links to CI-built packages/containers. <br />You can use this to test packages locally.</td>
  </tr>
</table>

For members of the Bioconda project, the following command is also available:

<table>
  <tr>
    <td><code>@BiocondaBot please merge</code></td>
    <td>Upload built packages/containers and merge a PR. <br />Someone must approve a PR first! <br />This reduces CI build time by reusing built artifacts.</td>
  </tr>
</table>

Also, the bot watches for comments from non-members that include `@bioconda/<team>` and will automatically re-post them to notify the addressed `<team>`.

</details>

### CODE REVIEW

**Comment by jfallmann** [comment]
@BiocondaBot please add label

File: recipes/diego/build.sh [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+
+mkdir -p ${PREFIX}/bin
+
+chmod +x DIEGO/*.py
+chmod +x DIEGO/*.pl
+
+mv DIEGO/*.py ${PREFIX}/bin/
+mv DIEGO/*.pl ${PREFIX}/bin/
+
+touch LICENSE					# This has to be removed once the LICENSE file was added upstream, so far this information is only available at the website
```
Comment (eggzilla) [inline recipes/diego/build.sh:11]: Hi, could you please add the LICENSE either in the upstream repository or the full text of the GNU GPL v2.0?

Comment (jfallmann) [reply recipes/diego/build.sh:11]: Waiting for upstream

**Comment by dpryan79** [comment]
@bioconda-bot please merge

**Pull request merged**